force-app/main/default/classes/FieldSetController.cls

```diff
@@ -40,4 +40,13 @@ public with sharing class FieldSetController {
             throw Util.getAuraHandledException(e);
         }
     }
+
+    @AuraEnabled(cacheable=true)
+    public static Map<String, Object> getFieldSetByName(String objectName) {
```

Comment:
Can this please be plural? `getFieldSetsByObject` or `getFieldSetsByObjectName` ?

Reply:
This is actually one field set per field set name. but my local var is plural so I will change that to singular.